### DISCUSSION TITLE
PrimeOne Figma UI Kit - Poseidon-theme

### DISCUSSION THREAD

**Discussion by stephenmangion** [primefaces/community#606] [PrimeVue Templates] [answered]
I need some help - I have purchased the PrimeOne Figma UI Kit with the intention of creating Visual Designs based on the Poseidon Theme.

Is there a way to download a UI kit that is themed using the Poseidon Theme pls?

We would like to access the figma UI kit that is Poseidon-themed, and be able to create visual designs before handing it over to the development team.

Can you help pls?

TIA
Stephen

**Comment by tugcekucukoglu** [comment, marked as answer]
Poseidon uses Lara theme and Figma UI Kit is compatible with Lara.

Regards,

**Comment by AshishFreak** [comment]
@stephenmangion hi can you please share the theme ?
